### PR TITLE
Clarify zwave migration for zwave_js

### PR DESCRIPTION
**Pull request by arnom-ms**
I had a old config in which I still had the manual config entry in the configuration file. If you do not remove it, zwave simply comes back after reboot. This change makes the user aware of this case.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @arnom-ms, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20189).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -515,6 +515,8 @@ There is no automatic migration wizard for the `ozw` integration. Please follow
 
 2) Remove the Z-Wave integration from Home Assistant: Configuration --> Integrations --> Z-Wave (or OpenZWave) --> Press the three dots and click Delete.
 
+    <div class='note info'>If you have configured Z-Wave manually, make sure to also remove the `zwave:` section from your configuration.yaml.</div>
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_js.markdown]: Note sure how we can get the `zwave:` to render as a codeblock. It doesn't seem to work when the note div is indented in a numbered list.

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -514,7 +514,8 @@ There is no automatic migration wizard for the `ozw` integration. Please follow
    <div class='note info'>Make a list of what node ID belongs to each device. Your network (Nodes and their config etc) is stored on the stick but the names you gave your devices and entities are not. This step is optional but will save you a lot of time later.</div>
 
 2) Remove the Z-Wave integration from Home Assistant: Configuration --> Integrations --> Z-Wave (or OpenZWave) --> Press the three dots and click Delete.
-
+> Note: If you have configured 'zwave' manually, make sure to also remove it from the configuration.yaml.
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_js.markdown]:
```suggestion

    <div class='note info'>If you have configured Z-Wave manually, make sure to also remove the `zwave:` section from your configuration.yaml.</div>
```

File: source/_integrations/zwave_js.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/zwave_js.markdown]:
```suggestion
    <div class='note info'>

    If you have configured Z-Wave manually, make sure to also remove the `zwave:` section from your `configuration.yaml`.

    </div>
```

The Markdown parser needs an empty line in order to switch HTML/Markdown context.

**Review by frenck — APPROVED** [review]

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**